main.py

```diff
@@ -1,12 +1,243 @@
 import requests
+from video_store import VideoStore
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
+def print_divider():
+    print("\noOoOoOoOoOooOoOoOoOoOooOoOoOoOoOoOoOoOoOoOoOoOoO\n")
+
 def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    print_divider()
+    print("✨✨✨✨✨WELCOME TO RETRO VIDEO STORE✨✨✨✨✨")
+    
+def list_options():
+
+    options = {
+        "1": "Create a new video record", 
+        "2": "Update a video record",
+        "3": "Delete a video record", 
+        "4": "Get information for all videos", 
+        "5": "Select a video", 
+        "6": "Create a new customer record",
+        "7": "Update a customer record",
+        "8": "Delete a customer record",
+        "9": "Select a customer",
+        "10": "Get information for all customers",
+        "11": "Check out a video to a customer",
+        "12": "Check in a video from a customer",
+        "13": "List all options",
+        "14": "Quit"
+        }
+
+    print_divider()
+    print("These are the actions you can perform: ")
+    print_divider()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_divider()
+
+    return options
+
+def make_choice(options, video_store):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Select 13 to see all options again or 14 to quit")
+        print("")
+        choice = input("Make your selection using the option number: ")
+        print("")
+
+    if choice in ['2','3'] and video_store.selected_video == None:
+        print("You must select a video before updating it or deleting it")
+        print("Please select a video!")
+        choice = "5"
+
+    elif choice in ['7','8'] and video_store.selected_video == None:
+        print("You must select a customer before updating them or deleting them")
+        print("Please select a customer!")
+        choice = "9"
+    
+    return choice
+
+def run_cli(play=True):
+
+    #initialize video_store
+    video_store = VideoStore(url="https://retro-video-store-api.herokuapp.com")
+    
+    # print choices
+    options = list_options()
+
+    while play==True:
+
+        # get input and validate
+        choice = make_choice(options, video_store)
+
+        video_store.print_selected()
+
+##### VIDEO CHOICES #####
+        if choice == '1':
+            print("Creating a new video record: \n")
+            title = input("What is the title of your video? ")
+            release_date = input("When was the release date of the video? ")
+            total_inventory = input("How many videos with this title are in stock? ")
+            response = video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_divider()
+            print("New video record created: \n", response["video"])
```

Comment:
This does not match the format of the response from the API we provided.  This issue shows up multiple places throughout the project.

```suggestion
            print("New video record created: \n", response)
```